scripts/hiseqx/xdemuxtiles.bash

```diff
@@ -60,6 +60,13 @@ FC=$( basename "$(basename "${RUNDIR}"/)" | awk 'BEGIN {FS="/"} {split($(NF-1),a
 # is this a dual-index FC?
 IS_DUAL=$(grep IndexRead2 "${RUNDIR}/runParameters.xml" | sed 's/<\/IndexRead2>\r//' | sed 's/    <IndexRead2>//')
 
+# calculate the number of lanes
+LANE_COUNT=$(awk '$0 ~/FlowcellLayout/ {split($0,arr," "); split(arr[2],o,"="); print o[2]}' "${RUNDIR}/RunInfo.xml" | sed 's/\"//g')
+declare -a lanes=()
+for (( i=1; i<=LANE_COUNT; i++ )); do
+  lanes[$i-1]=$i;
+done
+
```

Comment:
👍🏻 , also 😱 

and a little bit of 🏴‍☠️

Reply:
hehe, I was considering doing it in the demux module, but thought it would be good to have it were it was in the end. I could add descriptions of what the operations are doing?